addons/account/wizard/account_move_reversal.py

```diff
@@ -50,7 +45,7 @@ def reverse_moves(self):
         default_values_list = []
         for move in moves:
             default_values_list.append({
-                'ref': _('Reversal of: %s, %s') % (move.name, self.reason),
+                'ref': _('Reversal of: %s, %s') % (move.name, self.reason) if self.reason else _('Reversal of: %s') % (move.name),
```

Comment:
Translate one time `Reversal of`
`'ref': _('Reversal of: %s') % (('%s, %s' % (move.name, self.reason)) if self.reason else move.name, )`